src/components/map-page-components/map-sidebar/MapSidebar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Let's call this MapSidebarContent for clarity?

Reply:
See comment about sidebar on MapPage